airflow/providers/google/cloud/triggers/kubernetes_engine.py

```diff
@@ -306,18 +306,24 @@ async def run(self) -> AsyncIterator[TriggerEvent]:  # type: ignore[override]
             pod = await self.hook.get_pod(name=self.pod_name, namespace=self.pod_namespace)
         if self.do_xcom_push:
             await self.hook.wait_until_container_complete(
-                name=self.pod_name, namespace=self.pod_namespace, container_name=self.base_container_name
+                name=self.pod_name,
+                namespace=self.pod_namespace,
+                container_name=self.base_container_name,
+                poll_interval=self.poll_interval,
```

Comment:
This will not work with older version of K8s provider


We need to place this under version check like:
https://github.com/apache/airflow/blob/170b9ce90793191f7c97c6dbd09adab492082970/airflow/providers/google/cloud/operators/kubernetes_engine.py#L922-L928

Reply:
Thank you, updated.